szablon.Rmd

```diff
@@ -29,13 +31,13 @@ w wariancie "a"- dane według wyników z Powszechnego Spisu Rolnego 1996,
 w wariancie "b"- dane według wyników z Narodowego Spisu Powszechnego Ludności 
 i Mieszkań oraz Powszechnego Spisu Rolnego 2002
 
-Ustalmy wariant, który weźmiemy pod uwagę ora zakres lat, dla których 
+Ustalmy wariant, który weźmiemy pod uwagę oraz zakres lat, dla których 
 przeprowadzimy analizę.
 
 ```{r}
 bezrobocie_typ <- "b"
-rokStart <- 1990
```

Comment:
E, tu sobie coś sprawdzałem i niepotrzebnie to zmieniłem, domyślnie mogło być w sumie 1990.